msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -29,9 +34,12 @@ The following table shows a list of Teams conversation update events, with links
 | channel created     | channelCreated    | OnTeamsChannelCreatedAsync | [A channel was created](#channel-created) | Team |
 | channel renamed     | channelRenamed    | OnTeamsChannelRenamedAsync | [A channel was renamed](#channel-renamed) | Team |
 | channel deleted     | channelDeleted    | OnTeamsChannelDeletedAsync | [A channel was deleted](#channel-deleted) | Team |
-| team members added   | teamMemberAdded   | OnTeamsMembersAddedAsync   | [A Member added to team](#team-members-added)   | All |
-| team members removed | teamMemberRemoved | OnTeamsMembersRemovedAsync | [A Member was removed from team](#team-members-removed) | groupChat & team |
+| channel restored    | channelRestored    | OnTeamsChannelRestoredAsync | [A channel was restored](#channel-deleted) | Team |
+| members added   | membersAdded   | OnTeamsMembersAddedAsync   | [A member added](#team-members-added)   | All |
+| members removed | membersRemoved | OnTeamsMembersRemovedAsync | [A member was removed](#team-members-removed) | groupChat & team |
```

Comment:
```Suggestion
| members removed | membersRemoved | OnTeamsMembersRemovedAsync | [A member was removed](#team-members-removed) | groupChat and Team |
```

Reply:
This is incorrect, a "team" is the organizational construct inside "Teams".

Reply:
Thank you for the clarification. We will go with the original content 'team'.